tests/e2e/basic/client_connectivity_test/test_bridge_mode.py

```diff
@@ -1,4 +1,7 @@
 import pytest
+
+pytestmark = [pytest.mark.client_connectivity_test, pytest.mark.bridge]
```

Comment:
What does this do? Documentation

Reply:
this is a module level markering, as a part of pytest framework documentation and best practices, i am mentioning the ways to marker the test cases in an standard formate